docs/streams/upgrade-guide.html

```diff
@@ -120,6 +120,11 @@ <h3><a id="streams_api_changes_260" href="#streams_api_changes_260">Streams API
         We added a <code>--force</code> option in StreamsResetter to force remove left-over members on broker side when long session time out was configured
         as per <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-571%3A+Add+option+to+force+remove+members+in+StreamsResetter">KIP-571</a>.
     </p>
+    <p>
+        We added a <code>Suppressed.withLoggingDisabled()</code> and <code>Suppressed.withLoggingEnabled(config)</code>
+        method to disables (no s) and allows for configuration of the changelog topic
```

Comment:
`method[s]` (plural)

`to disables (no s)...` -> `to allow disabling or configuring of the changelog topic`